.github/workflows/deploy-prod.yml

```diff
@@ -0,0 +1,58 @@
+name: Deploy prod
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - production
+
+jobs:
+  deploy:
+    if: github.repository == 'fluid-project/lcil'
+
+    runs-on: ubuntu-latest
+
+    concurrency: deploy-prod
+
+    steps:
+      - name: Start deployment
+        uses: bobheadxi/deployments@v0.6.0
+        id: deployment
+        with:
+          step: start
+          token: ${{ secrets.GITHUB_TOKEN }}
+          env: production
+
+      - name: Deploy stack
+        uses: appleboy/ssh-action@v0.1.5
+        with:
+          host: ${{ secrets.SSH_HOSTNAME }}
+          username: ${{ secrets.SSH_USERNAME }}
+          key: ${{ secrets.SSH_PRIVATE_KEY }}
+          port: ${{ secrets.SSH_PORT }}
+          proxy_host: ${{ secrets.SSH_PROXY_HOSTNAME }}
+          proxy_username: ${{ secrets.SSH_PROXY_USERNAME }}
+          proxy_key: ${{ secrets.SSH_PROXY_PRIVATE_KEY }}
+          proxy_port: ${{ secrets.SSH_PROXY_PORT }}
+          script: |
+            cd /srv/fluid-lcil-prod && \
+            git checkout -f && \
+            git checkout production && \
+            git pull && \
+            ./bin/deploy-cloud.sh
```

Comment:
The deploy script will be different between dev and production. Specifically for the seeding and migration so that it doesn't wipe the database on deploy. See: https://github.com/fluid-project/lcil#migrating-and-seeding The seeding really just needs to happen once, but the seeder will skip actually seeding if the tables are already populated.